packages/babel-plugin-transform-class-properties/test/fixtures/loose/constructor-collision-ignores-types/expected.js

```diff
@@ -0,0 +1,9 @@
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
```

Comment:
The original version of this test compiled to a class -- I'd like to keep that behavior as I'm not sure if the issue reproduces when compiling all the way down to a function.